QUEST_LV_0100.tsv

```diff
@@ -4723,15 +4723,15 @@ QUEST_LV_0100_20150317_004722	Pour the sources of the magic of the Mausoleum to
 QUEST_LV_0100_20150317_004723	The protector of the secret told you that the Mausoleum of Zachariel was actually made to defeat Rexipher according to the forecast by the Goddess. Recharge the sources of the magic energy of the Mausoleum to the prepared pot.
```

Comment:
protector of the secret -> secret guardian
Mausoleum -> Royal Mausoleum
Zachariel -> the Great King Zachariel